akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -134,7 +134,7 @@ class PhiAccrualFailureDetector(
         val interval = timestamp - latestTimestamp
         // don't use the first heartbeat after failure for the history, since a long pause will skew the stats
         if (isAvailable(timestamp)) {
-          if (interval >= (acceptableHeartbeatPauseMillis / 2) && eventStream.isDefined)
+          if (interval >= (acceptableHeartbeatPauseMillis / 3 * 2) && eventStream.isDefined)
```

Comment:
Pretty arbitrary number – why?

Reply:
Ok, found the reason https://github.com/akka/akka/pull/25237#discussion_r196421342